RELEASING.md

```diff
@@ -41,9 +41,12 @@ The first step will create a source package from DART git checkout in the local
 system and upload it to Open Robotics PPA.
 
 **One Ubuntu distribution needs to be released at a time**. The information
-about target distribution (start with bionic) goes only in the Changelog entry. To
+about target distribution (bionic) goes only in the Changelog entry. To
 release multiple distribution, repeat this step 1 changing the changelog entry 
-(focal is also supported) and uploading the new source package to the PPA.
+and uploading the new source package to the PPA.
```

Comment:
If we are only doing bionic, do we need to "repeat this step ..."?

Reply:
Simplified the whole thing in 65f7cbd